src/Color/ColorConverter.php

```diff
@@ -173,7 +173,7 @@ private function convertPlain($color, array &$PDFAXwarnings = [])
 		$c = false;
 
 		if (preg_match('/^[\d]+$/', $color)) {
-			$c = [static::MODE_GRAYSCALE, $color]; // i.e. integer only
+			$c = [static::MODE_GRAYSCALE, (float) $color]; // i.e. integer only
```

Comment:
Given the regexp pattern (and the comment), float does not make sense.

Reply:
@finwe You're right, but `return float[]` also makes no sense if a string is returned. All other methods also return a float. If it makes more sense for you to convert to `int` at this point, then I would also agree.